### PR TITLE
Make peg-solitaire ready for reflex-dom-0.4

### PR DESCRIPTION
**Pull request by hansroland**
Please check changes in file *default.nix*. Thanks!

### CODE REVIEW

File: peg-solitaire/src/Main.hs [hunk 1]
```diff
@@ -22,7 +23,7 @@ data Trool = Yes | Idk | No deriving (Eq)
 
 type Board = Array Point Trool
 
--- | If the user chooses an amibuous starting point we need to remember it
+-- | If the user chooses an amibguous starting point we need to remember it
```
Comment (matthewbauer) [inline peg-solitaire/src/Main.hs:26]: lol

File: peg-solitaire/src/Main.hs [hunk 2]
```diff
@@ -56,7 +57,7 @@ main = mainWidgetWithCss ( $(embedFile "static/css/normalize.css")
         rec gs  <- foldDyn move initialState pos
             pos <- mkBoard gs
             elAttr "h3" ("style" =: "text-align: center") $
-                    dynText =<< mapDyn (\g -> "Score: " ++ (show $ score g)) gs
+                    dynText =<< (return . fmap (\g -> "Score: " <> ((T.pack . show) $ score g))) gs
```
Comment (ElvishJerricco) [inline peg-solitaire/src/Main.hs:60]: Why not just `dynText $ fmap (\g -> "Score: " <> (T.pack . show $ score g)) gs`?

Or if you like point free ;) `dynText $ fmap (mappend "Score: " . T.pack . show . score) gs`?

File: peg-solitaire/src/Main.hs (hunk 1 above)
Comment (hansroland) [reply peg-solitaire/src/Main.hs:26]: Thanks for fixing this typo!

File: peg-solitaire/src/Main.hs (hunk 2 above)
Comment (hansroland) [reply peg-solitaire/src/Main.hs:60]: Thank you for improving my code. Question: Do I have to take some additional actions before this PR can be merged?

**Comment by luigy** [comment]
Thanks for this PR #17 will include support using latest reflex* pkgs

**Pull request closed without merging**